### DISCUSSION TITLE
Code of Conduct?

### DISCUSSION THREAD

**Discussion by mattdm** [StractOrg/stract#151] [General]
I notice this seems absent from the contributor docs. I know from hard experience that it's better to have this in place before you need to worry about it.

If you haven't considered this, I recommend the Contributor Covenant as a good, commonly-used and generally understood option.

https://www.contributor-covenant.org/version/2/1/code_of_conduct/

**Comment by mikkeldenker** [comment]
Thank you, this is simply an oversight on my part. I'll get a code of conduct added right away.
